### PR TITLE
Thomas Cavanagh - Okta application edit

### PR DESCRIPTION
**Pull request by WatertonTC**
## Description:
- In Index file:
    - Fixed a link style which links to another Okta Developer page so that it has the internal style.
    - Added a link to a multifactor sign-in experience guide. 
    - Edited some heading overview sentences for clarity and consistency. 
    - Removed extra white space under the **What you need** heading. 
- In Configure your app file:
    - Deleted extra line. 
- In Get user profile file:
    - Added **Optional**  to the heading and changed it to an H3.
    - Added an anchor link to a previous heading. 
-  In Integration steps file:
    - Added a period to alt text sentence. 
    - Edited the sentence under the **Your app displays...** heading for clarity. 
        - Added a sentence to introduce the image below.
    - Changed the use of **Submit** in the **User submits their username and password** heading and sentence below it. 
    - Edited the wording of the authentication status headings for consistency and descriptiveness. 
    - Separated a two clause sentence for enhanced clarity. 
    - Edited the sentence under the **Handle other statuses** heading for clarity. 
- In Password optional file:
    - Combined the nested notes into one.
- In Summary of steps file:
    - Added a period to alt text sentence. 
    - Edited the wording of the introductory sentence for consistency.

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md](https://github.com/okta/okta-developer-docs/blob/a60e4447f72f947fffd7c35103025740e6ec209c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Findex.md) | [94](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c1f55e30-00d8-4250-869f-dcc7e8d6ba16) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/configureyourapp.md](https://github.com/okta/okta-developer-docs/blob/a60e4447f72f947fffd7c35103025740e6ec209c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Fnodejs%2Fconfigureyourapp.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/78a69611-fc5d-44cb-8df0-86a7e4538c9e) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/getuserprofile.md](https://github.com/okta/okta-developer-docs/blob/a60e4447f72f947fffd7c35103025740e6ec209c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Fnodejs%2Fgetuserprofile.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/feef01c8-81c4-4010-8b0c-89609ad6b5fd) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/integrationsteps.md](https://github.com/okta/okta-developer-docs/blob/a60e4447f72f947fffd7c35103025740e6ec209c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Fnodejs%2Fintegrationsteps.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/16f0c532-c2e0-4314-acc2-3d04fa981784) | 69  | 69  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/pwdoptionalusecase.md](https://github.com/okta/okta-developer-docs/blob/a60e4447f72f947fffd7c35103025740e6ec209c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Fnodejs%2Fpwdoptionalusecase.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9802b41e-35a3-4101-a028-49e7a8186439) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/nodejs/summaryofsteps.md](https://github.com/okta/okta-developer-docs/blob/a60e4447f72f947fffd7c35103025740e6ec209c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foie-embedded-sdk-use-case-basic-sign-in%2Fmain%2Fnodejs%2Fsummaryofsteps.md) | [93](https://okta.acrolinx.cloud/api/v1/checking/scorecards/05595085-dc4d-4c4e-b2b8-1d0f7870df45) | 100  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.98cc4ff4-fdfc-4711-b1ff-c714ade36f5e?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)
